presto-main/src/main/java/com/facebook/presto/translator/RowExpressionTranslator.java

```diff
@@ -0,0 +1,33 @@
+package com.facebook.presto.translator;
+
+import com.facebook.presto.spi.ColumnHandle;
+import com.facebook.presto.spi.function.FunctionMetadataManager;
+import com.facebook.presto.spi.function.StandardFunctionResolution;
+import com.facebook.presto.spi.relation.DeterminismEvaluator;
+import com.facebook.presto.spi.relation.LogicalRowExpressions;
+import com.facebook.presto.spi.relation.RowExpression;
+import com.facebook.presto.spi.relation.VariableReferenceExpression;
+import com.facebook.presto.spi.translator.ExpressionTranslator;
+import com.facebook.presto.spi.translator.TargetExpression;
+
+import java.util.Map;
+
+public class RowExpressionTranslator<T>
```

Comment:
User uses this class in their ComputePushdown class (see JdbcComputePushdown.java) and calls translate on it.